markhor_control/config/tracks.yaml

```diff
@@ -1,43 +1,44 @@
-  diff_drive_controller:
-    type: "diff_drive_controller/DiffDriveController"
-    left_wheel: ['flipper_fl_motor_j', 'flipper_rl_motor_j']
-    right_wheel: ['flipper_fr_motor_j', 'flipper_rr_motor_j']
-    publish_rate: 50
-    pose_covariance_diagonal: [0.001, 0.001, 0.001, 0.001, 0.001, 0.03]
-    twist_covariance_diagonal: [0.001, 0.001, 0.001, 0.001, 0.001, 0.03]
-    cmd_vel_timeout: 0.25
-    velocity_rolling_window_size: 2
+  tracks:
+    # Diff drive controller
+    diff_drive_controller:
+      type: "diff_drive_controller/DiffDriveController"
+      left_wheel: ['flipper_fl_motor_j', 'flipper_rl_motor_j']
+      right_wheel: ['flipper_fr_motor_j', 'flipper_rr_motor_j']
+      publish_rate: 50
+      pose_covariance_diagonal: [0.001, 0.001, 0.001, 0.001, 0.001, 0.03]
+      twist_covariance_diagonal: [0.001, 0.001, 0.001, 0.001, 0.001, 0.03]
+      cmd_vel_timeout: 0.25
+      velocity_rolling_window_size: 2
 
-    wheel_separation : 0.46
-    wheel_radius : 0.125
+      wheel_separation : 0.46
+      wheel_radius : 0.125
 
-    # Base frame_id
-    base_frame_id: markhor_root
+      # Base frame_id
+      base_frame_id: markhor_root
 
-    # Odometry fused with IMU is published by robot_localization, so
-    # no need to publish a TF based on encoders alone.
-    enable_odom_tf: false
+      # Odometry fused with IMU is published by robot_localization, so
+      # no need to publish a TF based on encoders alone.
+      enable_odom_tf: false
 
-    # Hardware provides wheel velocities
-    estimate_velocity_from_position: false
+      # Hardware provides wheel velocities
+      estimate_velocity_from_position: false
 
-    # Wheel separation and radius multipliers
-    wheel_separation_multiplier: 1.0 # default: 1.0
-    wheel_radius_multiplier    : 1.0 # default: 1.0
+      # Wheel separation and radius multipliers
+      wheel_separation_multiplier: 1.0 # default: 1.0
+      wheel_radius_multiplier    : 1.0 # default: 1.0
 
-    # Velocity and acceleration limits
-    # Whenever a min_* is unspecified, default to -max_*
-    linear:
-      x:
-        has_velocity_limits    : true
-        max_velocity           : 1.5   # m/s
-        has_acceleration_limits: true
-        max_acceleration       : 1 # 3 #  m/s^2
-    angular:
-      z:
-        has_velocity_limits    : true
-        max_velocity           : 2.25   # rad/s
-        has_acceleration_limits: true
-        max_acceleration       : 4.5   # 6 # rad/s^2
+      # Velocity and acceleration limits
+      # Whenever a min_* is unspecified, default to -max_*
+      linear:
+        x:
+          has_velocity_limits    : true
+          max_velocity           : 1.5   # m/s
+          has_acceleration_limits: true
+          max_acceleration       : 1 # 3 #  m/s^2
+      angular:
+        z:
+          has_velocity_limits    : true
+          max_velocity           : 2.25   # rad/s
+          has_acceleration_limits: true
+          max_acceleration       : 4.5   # 6 # rad/s^2
```

Comment:
Line endings...

Reply:
Again, please try to be more clear about what you mean by the line ending. It is not clear to me what the issue is.